snuba/settings/__init__.py

```diff
@@ -84,9 +83,6 @@
 # DEPRECATED, please use KAFKA_BROKER_CONFIG instead
 STORAGE_BROKER_CONFIG: Mapping[str, Mapping[str, Any]] = {}
 
-# DEPRECATED, please use KAFKA_TOPIC_MAP instead
-STORAGE_TOPICS: Mapping[str, Mapping[str, Any]] = {}
-
```

Comment:
Was this already removed from the other environments ?

Reply:
No. It can only be removed after KAFKA_TOPIC_MAP is added in https://github.com/getsentry/snuba/pull/1820 to replace it.